### ISSUE TITLE
Find in page a11y 

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/android-components#3912]
Related issue [https://github.com/mozilla-mobile/fenix/issues/148](https://github.com/mozilla-mobile/fenix/issues/148)

The next/prev and close buttons should have larger touch targets.
